File: Comp-u-Lock-Web/app/models/computer.rb
class Computer < ActiveRecord::Base
  attr_accessible :user_id, :enviroment, :ip_address, :name, :account_attributes, :history_attributes

  belongs_to :user

  validates :name, presence: true
  validates :enviroment, presence:  true
  validates :user_id, presence: true

  has_many :account, dependent: :destroy
  has_many :history, dependent: :destroy

  accepts_nested_attributes_for :account, :history

  def as_json options={}
    {
      id: id,
      user_id: user_id,
      name: name,
      enviroment: enviroment,
      ip_address: ip_address,

      account_attributes: account,
      history_attributes: history, 

      created_at: created_at,
      update_at: updated_at
    }
  end
end

File: Comp-u-Lock-Desktop/Database/Models/Computer.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Database.Models
{
    public class Computer
    {
        public int Id { get; set; }

        public int UserId { get; set; }
        public string Name { get; set; }
        public string Enviroment { get; set; }
        public string IpAddress { get; set; }

        public virtual List<Account> Accounts { get; set; }

        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }

        public Computer()
        {

        }
    }
}
